docs/understand/architecture.md

```diff
@@ -21,10 +21,23 @@ lakeFS is distributed as a single binary encapsulating several logical services:
 
 The server itself is stateless, meaning you can easily add more instances to handle bigger load.
 
-lakeFS stores data in an underlying [S3 bucket](https://aws.amazon.com/s3/){:target="_blank"} with some of its metadata stored in [PostgreSQL](https://www.postgresql.org/){:target="_blank"}. (see [Data Model](data-model.md))
+lakeFS stores data in an underlying object store ([S3](https://aws.amazon.com/s3/), [GCS](https://cloud.google.com/storage) or [ABS](https://azure.microsoft.com/en-us/services/storage/blobs/)) store, with some of its metadata stored in [PostgreSQL](https://www.postgresql.org/){:target="_blank"} (see [Data Model](data-model.md)).
+
+<!-- The below draw.io diagram source can be found here: https://drive.google.com/file/d/1lctPtGVEmOlCNHi3jiW4XXmyQQFkxzyx/view?usp=sharing -->
 
 ![Architecture]({{ site.baseurl }}/assets/img/arch.png)
 
+## Ways to deploy lakeFS
+
+lakeFS releases includes [binaries](https://github.com/treeverse/lakeFS/releases) for common operating systems and a [containerized option](https://hub.docker.com/r/treeverse/lakefs).
+Check out our guides for running lakeFS on [K8S](../deploy/k8s.md), [ECS](../deploy/aws.md#on-ecs), [Google Compute Engine](../deploy/gcp.md#on-google-compute-engine) and [more](../deploy/).
+
+### Load Balancing
+
+lakeFS receives HTTP requests through 3 components: S3 Gateway, OpenAPI Gateway and the Frontend UI.
+lakeFS uses a single port that serves all 3 endpoints, so for most use-cases a single load-balancer pointing
+to lakeFS server(s) would do.
+
 ## lakeFS Components
 
 ### S3 Gateway
```

Comment:
This section's content starts with `The API Gateway implements lakeFS' compatibility with S3`.
Shouldn't it be `The S3 Gateway implements lakeFS' compatibility with S3`?

Reply:
It also says that `To achieve this, the gateway exposes an HTTP listener on a dedicated host and port.`
I guess that it's also out of date as stated in line 38 above?

Reply:
Right, removing this line